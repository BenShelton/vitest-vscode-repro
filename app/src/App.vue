<template>
  <main>
    <p v-text="text" />
    <p v-text="now" />
  </main>
</template>

<script lang="ts">
import { defineComponent, ref } from 'vue'
import { secondsBetween } from '@/utils/time'
import type { Text } from 'types/sample'

export default defineComponent({
  setup () {
    const text = ref<Text>('Hello')

    const now = secondsBetween(10000, 20000)

    return {
      text,
      now,
    }
  },
})
</script>
