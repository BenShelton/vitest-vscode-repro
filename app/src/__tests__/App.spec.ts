import { shallowMount } from '@vue/test-utils'
import { describe, it } from 'vitest'

import App from '@/App.vue'


describe('App', () => {
  async function createWrapper () {
    return {
      wrapper: shallowMount(App)
    }
  }

  it('should match snapshot', async ({ expect }) => {
    const { wrapper } = await createWrapper()
    expect(wrapper.html()).toMatchSnapshot()
  })
})
