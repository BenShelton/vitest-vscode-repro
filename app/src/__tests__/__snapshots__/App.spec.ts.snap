// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`App > should match snapshot 1`] = `
"<main>
  <p>Hello</p>
  <p>10</p>
</main>"
`;
